components/developer/gcc-5/patches/ld-flag.patch

```diff
@@ -0,0 +1,24 @@
+--- ./gcc/config/sol2.h.orig	2016-05-08 21:13:10.810423614 +0200
++++ ./gcc/config/sol2.h	2016-05-08 21:16:55.6815353 +0200
+@@ -195,8 +195,8 @@
+   "%{G:-G} \
+    %{YP,*} \
+    %{R*} \
+-   %{!YP,*:%{p|pg:-Y P,%R/usr/lib/libp%R/lib:%R/usr/lib} \
+-	   %{!p:%{!pg:-Y P,%R/lib:%R/usr/lib}}}"
++   %{!YP,*:%{p|pg:-Y P,%R/usr/gcc/5/lib:%R/lib:%R/usr/lib -R %R/usr/gcc/5/lib} \
++	   %{!p:%{!pg:-Y P,%R/usr/gcc/5/lib:%R/lib:%R/usr/lib -R %R/usr/gcc/5/lib}}}"
```

Comment:
-Y P has the same effect but restricts to the list of directories passed as argument. Adding -L would be redundant.